test/TEST-99-RESOLVED/test.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: LGPL-2.1-or-later
+set -e
+
+TEST_DESCRIPTION="Tests for systemd-resolved"
+TEST_NO_NSPAWN=1
```

Comment:
This is to isolate the network stuff I imagine? Would it be possible to add an option to run nspawn with `--private-network` instead? Qemu is soooo sloooow

Reply:
I'm not sure how well resolved & networkd run under nspawn, but it's definitely something I could check out later.

Reply:
they should work pretty well, it's the default for mkosi images

Reply:
It indeed works (so far) without issues and is definitely faster, so I switched it over to nspawn-only, thanks!